5_recognition.ipynb

```diff
@@ -14,18 +16,54 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Object detection by image recognition\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Objective</b>\n",
+    "    <p>In this notebook, we will learn how to calculate position of the object using image recognition.</p>\n",
```

Comment:
"    <p>In this notebook, we will learn how to calculate the position of an object using image recognition.</p>\n",